bundles/mapping/mapmodule/plugin/getinfo/GetInfoPlugin.js

```diff
@@ -311,10 +311,13 @@ Oskari.clazz.define(
                 .filter((layer) => layerIds.includes(layer.getId()))
                 .reduce((result, layer) => {
                     const params = layer.getParams();
-                    if (typeof params !== 'object' || !Object.keys(params).length) {
+                    if (typeof params !== 'object') {
                         return result;
                     }
+
+                    params.STYLES = layer.getCurrentStyle().getName();
                     result[layer.getId()] = params;
+
```

Comment:
Don't modify the params directly as it creates a side-effect that might very well change how the layer behaves after calling this:
```suggestion
                    result[layer.getId()] = {
                         ...params,
                         STYLES: layer.getCurrentStyle().getName()
                     };
```